.github/workflows/ci.yml

```diff
@@ -98,14 +103,14 @@ jobs:
   check:
     name: Rust check ${{ matrix.check }} (${{ matrix.rust-target }})
     needs: config
-    if: ${{ needs.config.outputs.check == 'true' }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ubuntu-latest]
         rust: [stable]
         rust-target: [x86_64-unknown-linux-gnu]
-        check: [fmt] ## [fmt, clippy] ignore clippy for now
+#        check: [fmt --all -- --check, clippy -- -D warnings]
+        check: [fmt --all -- --check, check --features runtime-benchmarks] # skip clippy for now
```

Comment:
Also check for compiler errors with the `--features runtime-benchmarks` we often forgot this.